class-instant-articles-publisher.php

```diff
@@ -93,10 +98,12 @@ public static function submit_article( $post_id, $post ) {
 
 				try {
 					// Import the article.
-					$client->importArticle( $article, $take_live );
+					$submission_id = $client->importArticle( $article, $take_live );
+					update_post_meta( $post_id, self::SUBMISSION_ID_KEY, $submission_id );
 				} catch ( Exception $e ) {
 					// Try without taking live for pages not yet reviewed.
-					$client->importArticle( $article, false );
+					$submission_id = $client->importArticle( $article, false );
+					update_post_meta( $post_id, self::SUBMISSION_ID_KEY, $submission_id );
```

Comment:
Does this try/catch works well?
Is it needed to try twice?
I know it was here before,  but looks weird.
Its ok to just file an issue so we can double check it later.